src/test/scala/org/neo4j/spark/Neo4jOptionsTest.scala

```diff
@@ -42,45 +42,6 @@ class Neo4jOptionsTest {
     assertEquals(NodeSaveMode.Overwrite, neo4jOptions.relationshipMetadata.targetSaveMode)
   }
 
-  @Test
```

Comment:
why you removed these?

Reply:
Because I added the same tests on the Writer and Reader IT